src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+    public Circle(String color , double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        super.draw();
```

Comment:
Ensure that the `super.draw()` call is necessary. This depends on the implementation of the `Figure` class. If `Figure` does not have a `draw()` method or if it does not need to be called, this line can be removed.